src/cc/frontends/clang/b_frontend_action.cc

```diff
@@ -354,8 +354,14 @@ bool BTypeVisitor::VisitCallExpr(CallExpr *Call) {
           txt += ", bpf_get_smp_processor_id(), " + args_other + ")";
         } else {
           if (memb_name == "lookup") {
```

Comment:
Ok, I was recalling the discussion about what operations on the table would be allowed, and I must have remembered wrong. I will add a new op (name suggestions?). Any thoughts on how to expose the flags?